src/Main.java

```diff
@@ -0,0 +1,47 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        int grade;
+
+        System.out.println("Gib eine Note zwischen 1 & 5 ein");
+        grade = sc.nextInt();
+
+        if(grade >= 1 && grade <= 5) {

```

Comment:
bitte nochmal autoformattieren und die deklaration & initialisierung von grade in einer zeile mergen.